ansible/roles/jetson_services/tasks/main.yml

```diff
@@ -3,12 +3,13 @@
       src: "files/{{ item }}.service"
       dest: "/etc/systemd/system/{{ item }}.service"
   with_items:
+      - rover-jetson-auton-filter
```

Comment:
nit: keep alpha order?

Reply:
Preference thing but not sure why we wouldn't

Reply:
fixed